pkgs/development/python-modules/treeo/default.nix

```diff
@@ -8,21 +8,18 @@
 
 buildPythonPackage rec {
   pname = "treeo";
-  version = "0.4.0";
+  # At the time of writing (2022-03-29) the latest version is 0.0.11 but that is
+  # incompatible with other packages, esp. treex. See https://github.com/cgarciae/treex/issues/68.
```

Comment:
```suggestion
  # 0.4.0 was an accidental release and was supposed to be 0.0.4
```
much more important

Reply:
Fixed in https://github.com/NixOS/nixpkgs/pull/166167/commits/31da3adfc2a8ac2c302c2809486abbfc5895ad10.